File: internal/content/testdata/bench/go/unique-paths.go
func unique_paths(m int, n int) int {
	dp := make([]int, n)
	for j := range dp {
		dp[j] = 1
	}
	for i := 1; i < m; i++ {
		for j := 1; j < n; j++ {
			dp[j] += dp[j-1]
		}
	}
	return dp[n-1]
}

File: internal/content/testdata/bench/rust/unique-paths.rs
fn unique_paths(m: i64, n: i64) -> i64 {
    let n = n as usize;
    let mut dp = vec![1i64; n];
    for _ in 1..m {
        for j in 1..n {
            dp[j] += dp[j - 1];
        }
    }
    dp[n - 1]
}
